seasmon_xr/accessors.py

```diff
@@ -363,6 +363,65 @@ def whitsvc(
 
         return ds_out
 
+    def whitswcv(
+        self,
+        nodata: Union[int, float],
+        srange: np.ndarray = None,
+        p: float = None,
+        robust: bool = True,
+    ) -> xarray.Dataset:
+        """
+        Apply whittaker with GCV optimization of S.
+
+        Args:
+            dim: dimension to use for filtering
```

Comment:
I'm assuming you got that from the other methods, where it is incorrect as there's no dim argument. You can remove it

Reply:
Yes exactly, but you are right!